src/layouts/dashboard/header.tsx

```diff
@@ -323,14 +324,16 @@ const Header = () => {
   const notificationDrawerState = useDisclosure();
   const createWorkspaceDialog = useDisclosure();
   const {
-    currentWorkspace,
-    workspaceDialog,
-    userWorkspacesRequest: {
-      data: userWorkspaces,
-      refetch: refetchUserWorkspaces,
+    workspaceInfos: {
+      currentWorkspace,
+      workspaceDialog,
+      userWorkspacesRequest: {
```

Comment:
instance a new hook for this info `userWorkspaces`.

Reply:
On `handleCancel` and `handleClose` we dont need invalidate this request `refetchUserWorkspaces()`

Reply:
Done